product_development.md

```diff
@@ -0,0 +1,96 @@
+# Product development
+
+## Documents & Artifacts
+
+### Kanban board
+
+The Kanban board is the main tool of communication between stakeholders and the different product development units.
+
+The Kanban board is owned by the product management team.
+
+Engineers are responsible for maintaining cards on the Kanban board once they have pulled them into the backlog. Stories are always assigned when they are in progress, but never while they are in the backlog.
+
+All work that is done in either product, design or engineering should be reflected on the board. Stakeholders may have tickets on the board too.
+
+### New feature tickets
+
+The requirements engineering and conception of new features is owned by the product and design team.
+
+### Growth opportunity
+
+Anything that can help a department (not only product related) to improve their performance and the performance of our business.
+
+Growth opportunities should align with our long term product vision.
+
+Depending on the complexity those opportunities might be developed in small increments prioritizing value delivery.
+
+### OKR initiatives
+
+Strategic initiatives developed with a department and prioritized in close coordination with an objective owner. Those initiatives could be features and also research tasks, surveys, content work and more.
```

Comment:
It should be clear what OKR initiatives refer to in our day to day work. ...on the top lane of the roadmap board or something like that.

Reply:
This might be subject to change. I believe we should mark all different ticket styles accordingly.

Reply:
But I'll add a line for the lane.